zulip/integrations/bridge_with_irc/irc_mirror_backend.py

```diff
@@ -84,7 +85,7 @@ def forward_to_irc(msg: Dict[str, Any]) -> None:
     def on_privmsg(self, c: ServerConnection, e: Event) -> None:
         content = e.arguments[0]
         sender = self.zulip_sender(e.source)
-        if sender.endswith("_zulip@" + self.IRC_DOMAIN):
+        if sender.endswith(self.nickname + "@" + self.IRC_DOMAIN):
```

Comment:
Should these be `endswith` checks, or exact checks?

Reply:
I have replaced this line with exact check, and have tested it working.